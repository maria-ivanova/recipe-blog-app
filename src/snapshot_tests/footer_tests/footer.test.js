import React from 'react';
import renderer from 'react-test-renderer';
import Routing from '../helpers/router.js';
import Footer from '../../components/footer.js';

jest.mock('../../components/logo.js', () => 'Logo');

describe('Footer Component', () => {
    it('should render layout', () => {
        const tree = renderer.create(
            <Routing>
                <Footer />
            </Routing>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})