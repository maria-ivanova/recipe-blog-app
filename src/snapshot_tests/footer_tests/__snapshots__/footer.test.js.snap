// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render layout 1`] = `
Array [
  <footer
    className="sec content_sec footer_sec"
  >
    <div
      className="container container"
    >
      <Logo />
      <p>
        Наслаждавай се на вкусната храна със съветите на Spatulas.bg.
        <br />
        Готви с хиляди кулинари от всички краища на страната!
      </p>
      <p>
        Свържете се с нас: 
        <a
          href="mailto:spatulas@info.bg"
        >
          spatulas@info.bg
        </a>
      </p>
    </div>
  </footer>,
  <section
    className="sec copyrights"
  >
    <div
      className="container container"
    >
      <div
        className="ib"
      >
        2020 Spatulas Всички права запазени
      </div>
    </div>
  </section>,
]
`;
